core/controllers/reader.py

```diff
@@ -1757,6 +1757,39 @@ def put(self, exploration_id):
         self.render_json(self.values)
 
 
+class CuratedExplorationValidationHandler(base.BaseHandler):
```

Comment:
Again, ditto. All of this functionality already exists in `ValidateExplorationsHandler` in story_editor, unless this is somehow different? In which case, try to use that with additional flags, if possible.

Reply:
Done.